pkgs/applications/misc/pdfarranger/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub
+, wrapGAppsHook, intltool
+, python3Packages, gtk3, poppler_gi
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pdfarranger";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "jeromerobert";
+    repo = pname;
+    rev = version;
+    sha256 = "1f8m8r81322i97wkqpmf7a4kiwnq244n6cnbldh03jc49vwq2kxx";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook intltool
+  ] ++ (with python3Packages; [
+    setuptools distutils_extra
+  ]);
+
+  buildInputs = [
+    gtk3 poppler_gi
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pygobject3
+    pypdf2
+  ];
+
+  # incompatible with wrapGAppsHook
+  strictDeps = false;
+
+  doCheck = false; # no tests
+
+  meta = with stdenv.lib; {
```

Comment:
You could import lib instead of stdenv.lib here

Reply:
done